examples/components/toast/toast.styles.scss

```diff
@@ -0,0 +1,17 @@
+@import '~styles/theme';
+.example {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+    padding: 1em;
+}
+
+.button{
+    background: $secondary-color;
+    color: #ffffff;
+    padding: 1em;
+    margin: 1em;
+    width: 50%;
+    cursor: pointer;
+}
```

Comment:
subline

Reply:
I didn't see my mistakes because the sass linter doesn't parse the example folder. 
//sass-lint.yml
files: include: 'src/**/*.scss'

Reply:
Can you fix it in this PR maybe ?